src/Acuminator/Acuminator.Utilities/Common/EnumerableExtensions.cs

```diff
@@ -143,6 +144,29 @@ private static IEnumerable<T> ConcatIterator<T>(T extraElement, IEnumerable<T> s
 			}
 		}
 
+		/// <summary>
+		/// Reverses <see cref="ImmutableArray{T}"/>. This is an optimization method which allows to avoid boxing.
+		/// </summary>
+		/// <typeparam name="TItem">Type of the item.</typeparam>
+		/// <param name="source">The source to act on.</param>
+		/// <returns/>
+		[DebuggerStepThrough]
+		[MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
@dropsonic I know, It's not too big though and is used in small number of places currently. 
But because this may change later I'll remove AggressiveInlining

Reply:
fixed